server/api/db/sqldb/db.py

```diff
@@ -2417,6 +2418,12 @@ def verify_project_has_no_related_resources(self, session: Session, name: str):
             name, feature_vectors, "feature_vectors"
         )
 
+        # verify project can be deleted in nuclio
```

Comment:
I think `_verify_project_has_no_external_resources` is a more suiting place for this check